packages/common/src/viemActions.ts

```diff
@@ -0,0 +1,43 @@
+import type {
+  WriteContractParameters,
+  Transport,
+  Chain,
+  Account,
+  WalletActions,
+  WalletClient,
+  WriteContractReturnType,
+} from "viem";
+import { getAction } from "viem/utils";
+import { writeContract } from "viem/actions";
+import { writeContract as mud_writeContract } from "./writeContract";
+import { sendTransaction as mud_sendTransaction } from "./sendTransaction";
+import { type ContractWrite } from "./getContract";
+
+export const sessionActions = <TChain extends Chain, TAccount extends Account>(
```

Comment:
I tend to prefer one definition/export per file when we've got functions like this, as it makes it easier to navigate around

if we feel like we want more granularity, we can drop all these files into an `actions` folder to mirror viem but I tend to defer this sort of organization until things get messy rather than trying to organize too early (easier to add organization later than it is to undo/redo organization)

Reply:
Also thinking about the naming of this, I wonder if this should be something like `transactionQueueActions`

Reply:
I'll make a file for each function with the name of the function.

> transactionQueueActions

Considering the content, I like the name. I'll use that name for this.